src/esm_runscripts/batch_system.py

```diff
@@ -554,6 +590,11 @@ def write_simple_runscript(config, cluster, batch_or_shell="batch"):
                     cluster
                 ]["next_submit"]
 
+                # extra entries for each subjob
+                post_run_commands = batch_system.get_post_run_commands(config)
```

Comment:
```suggestion
                post_run_commands = batch_system.get_commands(config, "post_run_commands")
```
TODO for @mandresm: change it to something like this.